docs/src/man/getting_started.md

```diff
@@ -371,6 +371,9 @@ true
 
 In the first cases, `[:A]` is a vector, indicating that the resulting object should be a `DataFrame`, since a vector can contain one or more column names. On the other hand, `:A` is a single symbol, indicating that a single column vector should be extracted.
 
+Note, that similarly to Base Julia you are not allowed to select rows nor columns using `Tuple`s, use `AbstractVector`s instead.
```

Comment:
I have shortened it (I decided to add it as people ask about it - so it seems it creates confusion for newcomers).

For me the general answer to all the questions like this would be "it works like Base Julia" period (this was the design goal we had ~1 year ago and now we have it) 😄.